app/frontend/src/components/JsonBuilder.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <v-data-table
+    class="json-builder"
+    :disable-pagination="true"
+    :headers="headers"
+    :hide-default-header="true"
+    :hide-default-footer="true"
+    :items="items"
+  >
+    <template v-slot:item.key="props">
+      <v-text-field placeholder="Key" single-line v-model="props.item.key" />
+    </template>
+    <template v-slot:item.value="props">
+      <v-text-field placeholder="Value" single-line v-model="props.item.value" />
+    </template>
+    <template v-slot:item.action="{ item }">
+      <v-tooltip top>
+        <template v-slot:activator="{ on }">
+          <v-btn @click="deleteItem(item)" fab class="mb-2" small text v-on="on">
+            <v-icon>delete</v-icon>
+          </v-btn>
+        </template>
+        <span>Add Key/Value Pair</span>
+      </v-tooltip>
+    </template>
+    <template v-slot:footer>
+      <v-container>
+        <v-row justify="end">
+          <v-tooltip top>
+            <template v-slot:activator="{ on }">
+              <v-btn color="info" @click="addItem()" class="mb-2" v-on="on">
+                <v-icon :left="$vuetify.breakpoint.smAndUp">playlist_add</v-icon>
+                <span v-if="$vuetify.breakpoint.smAndUp">Add Entry</span>
+              </v-btn>
+            </template>
+            <span>Add Key/Value Pair</span>
+          </v-tooltip>
+        </v-row>
+      </v-container>
+    </template>
+  </v-data-table>
+</template>
+
+<script>
+export default {
+  name: 'jsonbuilder',
+  data() {
+    return {
+      headers: [
+        { text: 'Key', value: 'key' },
+        { text: 'Value', value: 'value' },
+        { text: 'Action', value: 'action' }
+      ],
```

Comment:
Unused now that the table header is gone?

Reply:
This is still used internally in the table even if it isn't shown on the page. I tried removing it and the entire table broke horribly.